src/components/common/AgreementType.vue

```diff
@@ -22,21 +22,21 @@
     </template>
 
     <!-- EDIT SECTION -->
-    <template v-else-if="isTypeBcCcc">
+    <div v-if="isTypeBcUlcCompany" :class="{ 'invalid-section': showErrorSummary }">
       <v-checkbox
         v-for="(item, index) in getIncorporationAgreementDocuments"
+        v-model="agreementType"
+        class="ml-6 mt-0 agreement-option-list"
         :id="`agreement-type-${item.code}`"
-        class="agreement-option-list"
         :key="index"
-        v-model="agreementType"
         :value="item.code"
         @change="changeAgreementType()"
       >
         <template v-slot:label>
-          <div v-html="item.description" class="agreement-option" />
+          <div v-html="item.description" class="ml-6 py-4 agreement-option" />
         </template>
       </v-checkbox>
-    </template>
+    </div>
```

Comment:
Please attach screenshots to show what the new UI looks like (here in the PR or in the ticket).

Reply:
Screenshots have been attached to the PR

Reply:
Thanks - very helpful!